organisations/service.go

```diff
@@ -135,14 +135,36 @@ func (cd service) Write(thing interface{}) error {
 		}
 		queries = append(queries, parentQuery)
 	}
-	return cd.cypherRunner.CypherBatch(queries)
+
+	return processCypherBatchWrite(cd, queries)
+}
+
+func processCypherBatchWrite(cd service, queries []*neoism.CypherQuery) error {
```

Comment:
Looking at the `BatchCypherRunner` in `neoutils` and it would be pretty easy to move the code there. It also feels like it would make more sense for the error struct to be owned by `neoutils` as well. It wouldn't be a breaking change unless anyone using it was expecting a `NeoError`. I don't throw away the error so you can always get to the `NeoError` if you need it.